packages/core/src/components/icon/icon.md

```diff
@@ -4,15 +4,62 @@
     See [**Icons**](#icons) for a searchable list of all available UI icons.
 </div>
 
+<div class="pt-callout pt-intent-primary pt-icon-info-sign">
+    <h5>SVG icons in 2.0</h5>
+    Blueprint 2.0 introduced SVG icon support throughout the ecosystem, and moved icon resources to a separate
+    __@blueprintjs/icons__ package. The `Icon` component now renders SVG paths and the icon classes are no longer
+    used by any Blueprint React component. Icon font support has been preserved but should be considered legacy,
```

Comment:
"... but should be considered a legacy feature that will be removed in a future major version."